stripe/_stripe_object.py

```diff
@@ -23,6 +23,9 @@
 from stripe import _util
 
 from stripe._stripe_response import StripeResponse, StripeStreamResponse
+import warnings
+
+_util.deprecated
```

Comment:
nit: rem?

Reply:
oh, good catch. This was for catching the circular import